src/views/InteractiveMap/ReactionDialog.vue

```diff
@@ -191,12 +191,12 @@ export default Vue.extend({
     },
     direction() {
       if (this.lowerBound >= 0) {
-        return " -> ";
+        return " 🠖 ";
       }
       if (this.upperBound <= 0) {
-        return " <- ";
+        return " 🠔 ";
       }
-      return " ⇌ ";
+      return " ⇄ ";
```

Comment:
Im not sure if this requires a change, but this sign is a bit different
than that returned by bigg api (encoded as `&#8652;` &#8652;)
https://api.dd-decaf.eu/bigg/models/universal/reactions/HPT_h

On the other hand metanetx api returns simple =